.github/workflows/ci.yml

```
@@ -616,10 +616,15 @@ jobs:
         uses: ./.github/actions/run-maven
         with:
           parameters: -f build-tools/pom.xml install
+      - name: Build optimize dependencies
+        uses: ./.github/actions/run-maven
+        with:
+          parameters: -pl optimize/backend -am install -Dquickly -Dskip.fe.build -PskipFrontendBuild -Dskip.docker
+          threads: .5C
       - name: Test
         uses: ./.github/actions/run-maven
         with:
-          parameters: -f optimize/pom.xml test -Dskip.fe.build -Dskip.docker
+          parameters: -f optimize/pom.xml  test -Dskip.fe.build -PskipFrontendBuild -Dskip.docker
```

```suggestion
          parameters: -f optimize/pom.xml test -PskipFrontendBuild -Dskip.docker
```

The profile already sets the `-Dskip.fe.build` to true, so you can remove that